### PR TITLE
[Docenhancement][Requestposting]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
Added note for activity payload. Please review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/build-adaptive-card-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/475636d117232f587145f36a8c32fcd9232907dd/msteams-platform/tabs/how-to/build-adaptive-card-tabs.md) | 92 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/36a7d91d-040f-41ec-aaa1-f1a4d72c2342) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/build-adaptive-card-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a1f2ebdf607c8dcdfc4cdb237747a777534b6dd2/msteams-platform/tabs/how-to/build-adaptive-card-tabs.md) | 92 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/125e2a30-e614-4836-8939-bb2a6eb3f726) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/tabs/how-to/build-adaptive-card-tabs.md [hunk 1]
```diff
@@ -94,6 +94,9 @@ Following is a sample Adaptive Card tab manifest:
 
 Communication between your Adaptive Card tab and your bot is done through `invoke` activities. Each `invoke` activity has a corresponding **name**. Use the name of each activity to differentiate each request. `tab/fetch` and `tab/submit` are the activities covered in this section.
 
+> [!NOTE]
+> All requests sent to [service URL](/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#base-uri&preserve-view=true) of your bot will be received as part of `activity` payload.
```
Comment (Wajeed-msft) [inline msteams-platform/tabs/how-to/build-adaptive-card-tabs.md:98]:
```suggestion
> Bot needs to send all the responses to [service URL](/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#base-uri&preserve-view=true).  Service URL is received as part of incoming `activity` payload. 
```

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/build-adaptive-card-tabs.md](https://github.com/MicrosoftDocs/msteams-docs/blob/14b846021c4aa06b0ea6075e5f7bd7b55dadc1cb/msteams-platform/tabs/how-to/build-adaptive-card-tabs.md) | 92 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0e20d3f9-31d4-4182-aada-92ff032fdeb0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**